Resolve ESLint warnings in src/index.js

### Overview
This file has ESLint errors or warnings:

```
src/index.js
  8:1  warning  'react-jss' import is duplicated  no-duplicate-imports
  9:1  warning  'react-jss' import is duplicated  no-duplicate-imports
```

### Action Items
Resolve the errors or warnings. If there is a good reason to keep any piece of offending code, disable ESLint as shown below.

### Resources/Instructions
To see all ESLint errors and warnings, run this from the command line: `npm run lint`
Or for only one file, run this: `npx eslint path/to/file.js`

To close this issue by disabling errors or warnings when there is a reason that they cannot be resolved, use comments. 
To disable a rule for the whole file, put this at the top of the file:
`/* eslint-disable rule-name */`
To disable a rule for one line, put this right before the line:
`// eslint-disable-next-line rule-name`
Or put this at the end of the line:
`// eslint-disable-line rule-name`
Additional commenting options can be found in the [ESLint docs](https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments).

### Dependency
This issue is one of many new issues this morning intended to replace the multi-file issue #169 with one issue per file.

--- a/src/index.js
+++ b/src/index.js
@@ -1,73 +1,74 @@
-import React from 'react'
-import ReactDOM from 'react-dom'
-import App from './App'
-import * as serviceWorker from './serviceWorker'
-import { ThemeProvider as RjssThemeProvider } from 'react-jss'
-import jss from 'jss'
-import preset from 'jss-preset-default'
-import { SheetsRegistry } from 'react-jss'
-import { JssProvider } from 'react-jss'
-import { BrowserRouter } from 'react-router-dom'
-import WebFont from 'webfontloader'
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+import {
+  ThemeProvider as RjssThemeProvider,
+  SheetsRegistry,
+  JssProvider,
+} from 'react-jss';
+import jss from 'jss';
+import preset from 'jss-preset-default';
+import { BrowserRouter } from 'react-router-dom';
+import WebFont from 'webfontloader';
+import {
+  createMuiTheme,
+  ThemeProvider as MuiThemeProvider,
+} from '@material-ui/core/styles';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import themeMui from './theme-mui';
 
-import { createMuiTheme, ThemeProvider as MuiThemeProvider } from '@material-ui/core/styles'
-import CssBaseline from '@material-ui/core/CssBaseline'
-import themeMui from './theme-mui'
-
-const theme = createMuiTheme(themeMui)
+const theme = createMuiTheme(themeMui);
 
 WebFont.load({
   google: {
-    families: [
-      'Work Sans:400,700',
-      'Source Code Pro:400',
-    ],
+    families: ['Work Sans:400,700', 'Source Code Pro:400'],
   },
-})
+});
 
 const setupJss = () => {
-  jss.setup(preset())
+  jss.setup(preset());
 
-  const sheetsRegistry = new SheetsRegistry()
+  const sheetsRegistry = new SheetsRegistry();
 
-  const globalStyleSheet = jss.createStyleSheet(
-    {
+  const globalStyleSheet = jss
+    .createStyleSheet({
       '@global': {
         body: {
           margin: '0',
         },
       },
-    }
-  ).attach()
+    })
+    .attach();
 
-  sheetsRegistry.add(globalStyleSheet)
+  sheetsRegistry.add(globalStyleSheet);
 
-  return sheetsRegistry
-}
+  return sheetsRegistry;
+};
 
-const sheets = setupJss()
+const sheets = setupJss();
 
 const themeR = {
-  primaryFontFamily: 'Work Sans, san-serif',
-  secondaryFontFamily: 'Space Mono, monospace',
-  thirdFontFamily: 'Source Code Pro, monospace',
-  primaryTextColor: 'black',
-  secondaryTextColor: 'white',
-  footerBG: '#242424',
-  headerBG: '#D8D8D8',
-  headerBG2: '#F8F8F8',
-  headerCircleBG: '#F4F4F4',
-  button1BG: '#0CB2E7',
-  button1TextColor: 'white',
-  button2BG: '#FFFFFF',
-  button2TextColor: '#979797',
   backgroundColor1: '#FFFFFF',
   backgroundColor2: '#F2F2F2',
   backgroundColor3: '#E1E1E1',
   backgroundColor4: '#242424',
   backgroundColor5: '#E9E9E9',
   backgroundColor6: '#C4C4C4',
-}
+  button1BG: '#0CB2E7',
+  button1TextColor: 'white',
+  button2BG: '#FFFFFF',
+  button2TextColor: '#979797',
+  footerBG: '#242424',
+  headerBG: '#D8D8D8',
+  headerBG2: '#F8F8F8',
+  headerCircleBG: '#F4F4F4',
+  primaryFontFamily: 'Work Sans, san-serif',
+  primaryTextColor: 'black',
+  secondaryFontFamily: 'Space Mono, monospace',
+  secondaryTextColor: 'white',
+  thirdFontFamily: 'Source Code Pro, monospace',
+};
 
 ReactDOM.render(
   <React.StrictMode>
@@ -83,11 +84,12 @@ ReactDOM.render(
     </MuiThemeProvider>
   </React.StrictMode>,
   document.getElementById('root')
-)
+);
 
 /*
  * If you want your app to work offline and load faster, you can change
  * unregister() to register() below. Note this comes with some pitfalls.
  * Learn more about service workers: https://bit.ly/CRA-PWA
  */
-serviceWorker.unregister()
+
+serviceWorker.unregister();

